lib/auth.ts

```diff
@@ -118,3 +121,33 @@ export const validateTemporaryToken = async (token: string) => {
     return prismaErrors(error, null);
   }
 };
+
+/**
+ * Returns the URL for the page if the user is not logged in
+ * @param authRole string of the authRole from the current session
+ * @returns string for the url
+ */
+const loginUrl = (authRole: string): string => {
+  switch (authRole) {
+    case UserRole.Administrator:
+      return "/admin/login/";
+    case UserRole.ProgramAdministrator:
+    default:
+      return "/auth/login/";
+  }
+};
+
+/**
+ * Returns the URL for the page if the user is unauthorized
+ * @param authRole string of the authRole from the current session
+ * @returns string for the url
+ */
+const unauthorizedUrl = (authRole: string): string => {
```

Comment:
To ensure typing we can instead use the `UserRole` as the type for `authRole`.
This ensures that only the UserRole enum is used.